src/PhpSpec/ObjectBehavior.php

```diff
@@ -130,6 +130,7 @@ public function offsetGet($key): Subject
      *
      * @param int|string $key
      */
+    #[\ReturnTypeWillChange]
```

Comment:
As mentioned in https://github.com/phpspec/phpspec/pull/1397#issuecomment-944081760, this comment was added instead of the `void` return type in order to preserve backwards compatibility.